public/assets/sass/components/_table.scss

```diff
@@ -1,6 +1,219 @@
+                transition: all .4s ;
+               }
+   
+            & > * {
+                align-self: center;
+                justify-self: center;
+                font-size: 1.4rem;
+                color: $white;
+                @include respond(small-phone){
+                    font-size: 1.1rem;                   
+                }
+            }
+
+            &-amount {
+                color: $green-money;
+            }
+
+            .container-none {display: none;}
+            @include respond(phone) {
+            .container-none {display: block;}
+           
+            .container {
+                grid-column: 1/-1;
+                position: relative;
+                transition: all .3s;
+                border-top: .2rem solid $gray-table-border;
+
+                & input {
+                    position: absolute;
+                    top: -1.9rem;
+                    right: .3rem;
+                    opacity: 0;
+                    cursor: pointer;
+                }
+                .arrow{
+                    width: 1.5rem;
+                    color: white;
+                    font-size: 1.5rem;
+                    transform: rotateZ(0deg);
+                    transition: all .3s;
+
+                }
+
+                &__arrow-box {
+                    width: 3rem;
+                    height: 2.5rem;
+                    position: absolute;
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+                    top: -3rem;
+                    right: .3rem;
+                    border-radius: .4rem;
+                    background-color: transparent;
+                    transition: all .3s;
+                    @include respond(small-phone){
+                        width: 2rem;
+                    }
+
+                    &:hover {
+                        background-color: $orange-arrow-box ;
+                        border: 1px solid $orange-arrow-box ;
+                        cursor: pointer;
+
+                        & .arrow {
+                            color: $black;
+                        }
+
+                        }
+                    }
+
+                .note-box {
+                    display: none;
+                    transition: all .8s;
+                    height: 0;
+    
+                }
+
+                & input:checked ~ .note-box{
+                    opacity: 1;
+                    grid-column: 1/-1;
+                    display: block;
+                    display: grid;
+                    visibility: visible;
+                    grid-template-columns: 1fr 4rem;
+                    grid-template-rows: minmax(9rem,min-content);
+                    height: 100%;
+                }
+                & input:checked + div .arrow {
+                    transform: rotateZ(180deg);
+                }
+
+            }
+                  
+               }
```

Comment:
Code indentation is important to parse the code snippets faster. Parentheses are not aligned properly, so I cannot trace the code blocks properly. Please, consider indenting and aligning your code blocks properly.